sam/data_sources/weather/regenradar.py

```diff
@@ -93,30 +102,50 @@ def read_regenradar(start_date, end_date, latitude=52.11, longitude=5.18, freq="
             "longitude={}, window={}"
         ).format(start_date, end_date, latitude, longitude, window)
     )
+
     if isinstance(start_date, str):
         start_date = _try_parsing_date(start_date)
     if isinstance(end_date, str):
         end_date = _try_parsing_date(end_date)
 
-    regenradar_url = "https://rhdhv.lizard.net/api/v3/raster-aggregates/?"
-    params = {
-        "agg": "average",
-        "rasters": "730d6675",
-        "srs": "EPSG:4326",
-        "start": str(start_date),
-        "stop": str(end_date),
-        "window": str(window),
-        "geom": "POINT+({x}+{y})".format(x=longitude, y=latitude),
-    }
-    params.update(kwargs)
-    params = "&".join("%s=%s" % (k, v) for k, v in params.items() if v is not None)
-
-    res = requests.get(regenradar_url + params, auth=(user, password))
-    res = res.json()
-    data = json_normalize(res, "data")
-
-    # Time in miliseconds, convert to posixct
-    data.columns = ["TIME", "PRECIPITATION"]
-    data["TIME"] = pd.to_datetime(data["TIME"], unit="ms")
+    date_range = pd.date_range(start_date, end_date, freq=batch_size)
+
+    result = []
+    for date in date_range:
+        start_date_, end_date_ = date, date + pd.Timedelta(batch_size)
+
+        print(start_date_, end_date_)
+
+        regenradar_url = "https://rhdhv.lizard.net/api/v3/raster-aggregates/?"
+        params = {
+            "agg": "average",
+            "rasters": "730d6675",
+            "srs": "EPSG:4326",
```

Comment:
This should probably be a parameter right? Or is this the only crs it accepts (also is it a typo or is it called srs here?)